protos/lightstreamer.proto

```diff
@@ -18,38 +18,38 @@ message BlockRange {
 message LightBlock {
 	uint32 protoVersion = 1;    // the version of this wire format, for storage
   uint64 sequence = 2;          // the height of this block
-  string hash = 3;             // the ID (hash) of this block, same as explorer
-  string previousBlockHash = 4;         // the ID (hash) of this block's predecessor
-  uint32 timestamp = 5;            // Unix epoch time when the block was mined
+  bytes hash = 3;             // the ID (hash) of this block, same as explorer
+  bytes previousBlockHash = 4;         // the ID (hash) of this block's predecessor
+  uint64 timestamp = 5;            // Unix epoch time when the block was mined
```

Comment:
Bug fix, was truncating timestamp when using uint32, not sure why but this fixed it.